check_zfs/metadata.yml

```diff
@@ -0,0 +1,13 @@
+package_name: fdgmon-check-zfs
+directory_name: check_zfs
+files:
+  - src: check_zfs.py
+    dest: usr/lib/monitoring/plugins/check_zfs
+    mode: 755
+version_git_autogenerated: true
+dependencies:
+  - python
```

Comment:
Is this script python2 only? If yes, I think it shouldn't be used anymore. If no, please use `python3` here. The `python`-package is for version 2.7